code/game/machinery/cryopod.dm

```diff
@@ -290,7 +290,10 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 
 	if(GLOB.announcement_systems.len)
 		var/obj/machinery/announcement_system/announcer = pick(GLOB.announcement_systems)
-		announcer.announce("CRYOSTORAGE", mob_occupant.real_name, announce_rank, list())
+		if(mob_occupant.job == JOB_NAME_CAPTAIN)
+			minor_announce("Captain [mob_occupant.real_name] has entered cryogenic storage.")
```

Comment:
Replace the word Captain in this string with [JOB_NAME_CAPTAIN]

Reply:
but wont it always be captain..

Reply:
oh go away with that-
this is why im ob break